zappa/wsgi.py

```diff
@@ -41,7 +41,6 @@ def create_wsgi_request(
     else:
         query = event_info.get("queryStringParameters", {})
         query_string = urlencode(query) if query else ""
-    query_string = unquote(query_string)
```

Comment:
remove import from file above

Reply:
@hellno I checked and `unquote` is still used here:

https://github.com/zappa/Zappa/blob/ee74dcfbaa2fe07a383a902464d7c14cb70da204/zappa/wsgi.py#L81